HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -25,16 +26,28 @@
 import com.microsoft.hsg.android.simplexml.HealthVaultInitializationHandler;
 import com.microsoft.hsg.android.simplexml.HealthVaultSettings;
 import com.microsoft.hsg.android.simplexml.client.HealthVaultClient;
+import com.microsoft.hsg.android.simplexml.client.HealthVaultRestClient;
 import com.microsoft.hsg.android.simplexml.client.RequestCallback;
 import com.microsoft.hsg.android.simplexml.things.types.types.PersonInfo;
 import com.microsoft.hsg.android.simplexml.things.types.types.Record;
+import com.microsoft.rest.ServiceResponse;
+
+import healthvault.client.implementation.MicrosoftHealthVaultRestApiImpl;
+import healthvault.client.models.ActionPlan;
+import healthvault.client.models.ActionPlansResponseActionPlanInstance;
+import okhttp3.ResponseBody;
+import retrofit2.Response;
+import rx.Observable;
+import rx.Subscriber;
+import rx.schedulers.Schedulers;
 
 public class MainActivity 
 	extends ListActivity 
 	implements HealthVaultInitializationHandler {
 
 	private HealthVaultApp service;
 	private HealthVaultClient hvClient;
+	private HealthVaultRestClient restClient;
```

Comment:
Removed

---
In reply to: [116320634](https://github.com/Microsoft/healthvault-java-sdk/pull/5#discussion_r116320634) [](ancestors = 116320634)